homeassistant/components/switch/rest.py

```diff
@@ -69,8 +69,9 @@ def is_on(self):
 
     def turn_on(self, **kwargs):
         """Turn the device on."""
+        headers = {'content-type': 'application/x-www-form-urlencoded'}
```

Comment:
```yaml
 headers:
    - name: Content-Type
...
```

Those would be handled as every other configuration entry. Take a look at the `def setup_platform():`